include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -190,6 +191,53 @@ class alignment_algorithm :
         return alignment_result{make_result(idx, first_range, second_range, cache)};
     }
 
+    /*!\brief Invokes the actual alignment computation given two sequences.
+     * \tparam    job_range_type  The type of underlying sequence pairs; must model std::ranges::view.
+     *
+     * \param[in] job_range   The range of underlying sequence pairs to be aligned.
+     *
+     * \returns A seqan3::alignment_result with the requested alignment outcomes.
+     *
+     * \details
+     *
+     * The algorithm always computes a pairwise alignment of two sequences over either a regular alphabet or
+     * packed alphabets in a SIMD vector. In the latter case an inter-vectorisation layout
+     * is used to compute l many pairwise alignments in parallel using special extended register instructions.
```

Comment:
What is `l`?

Reply:
@joergi-w this is ok. I have the clean up PR open. Depending on what will merged before I will then fix the documentation accordingly. It is also not accurate anymore.